includes/Core/Assets/Assets.php

```diff
@@ -108,6 +108,20 @@ function( $tag, $handle ) {
 			2
 		);
 
+		add_filter(
+			'googlesitekit_font_families',
```

Comment:
While this is mentioned in the IB, it's not really necessary and makes it a bit more complicated. Let's keep it simple and allow the array to be filtered by providing the initial value to the filter directly.